academy/ibc/channels.md

```diff
@@ -0,0 +1,166 @@
+    TimeoutHeight:      timeoutHeight,
+    TimeoutTimestamp:   timeoutTimestamp,
+  }
+}
+```
+`Sequence` denotes the sequence number of the packet in the channel.
+
+`TimeoutTimestamp` and `TimeoutHeight` dictate the time before which the receiving module must process a packet. 
+
+
+![Packet flow](/academy/ibc/images/packetflow.png)
+
+**Success Case**
+
+In the first step of a successful packet flow, application A will send a packet (call `sendPacket`) to the application B. `SendPacket` can be triggered by a user, but applications can also trigger this as the result of some other application logic. 
+
+Core IBC A will commit the packet to its own state and the relayer can query this packet and send a `RecvPacket` message to core IBC B. Core IBC handles a number of verifications, including verifying that the packet was indeed sent by chain A, that the packet came in the correct order if it was sent over an ordered channel, that the state commitment proof is valid etc. If this verification step is successful, core IBC will then route the packet to application B.
+
+Note that core IBC is unopinionated about the actual content of the packet data, as this data is at this point just bytes. It is the responsibility of the applications on either end to marshal and unmarshal the data from and to the expected data structures on either side. This is also why application version negotiation as discussed above in the channel handshakes is important, as different versions of an application may result in different expected data structures on either end of the channel and application.
+
+After receiving the packet data from core IBC, application B will then marshal the data blob into the expected structure and apply the relevant application logic. In the case of an ICS20 token transfer, for example, this would entail the minting of the received tokens on chain B to the specified receiver user account. Application B will then send an `Acknowledgment` message to core IBC B, which will again commit it to its own state so it can be queried and sent by a relayer to core IBC A.
+
+<HighlightBox type="info">
+
+**Synchronous and Asynchronous Acknowledgements**
+
+Acknowledgements can either take place synchronously or asynchronously. What this means is that the `OnRecvPacket` callback has a return value `Acknowledgement` which is optional. 
+
+In the case of a synchronous `Acknowledgement`, the callback will return an `Acknowledgement` at the end of the process and relayer can query this `Acknowledgement` packet and relay immediately after the process has finished. This is useful in cases in which application A is expecting an `AckPacket` in order to initiate some application logic `OnAcknowledgePacket`. For example, the sending chain of an ICS20 token transfer will do nothing in the case of a successful `AckPacket`, but in the case where an error is returned, the sending chain will unescrow the previously locked tokens.
```

Comment:
As i understood, a use case for the Ack even when there's no specific application logic `OnAcknowledgePacket`is to remove the commitment proof from the store, to avoid cluttering with old data